spatial/curve/hilbert.go

```diff
@@ -4,6 +4,50 @@
 
 package curve
 
+import (
+	"errors"
+	"fmt"
+)
+
+var ErrUnderflow = errors.New("order cannot be less than 1")
+var ErrOverflow = errors.New("overflow int")
+
+// NewHilbert2D constructs a [Hilbert2D] of the given order. NewHilbert2D
+// returns [ErrOverflow] if the order would cause Len and Pos to overflow.
```

Comment:
"returns **[ErrOverflow]**" is not quite true.

Reply:
I added "(wrapped)"